guide/qceff-examples.rst

```diff
@@ -0,0 +1,184 @@
+.. _quantile tracer:
+
+QCEFF: Examples with the Lorenz 96 Tracer Model
+===============================================
+
+
+The Quantile-Conserving Ensemble Filter Framework (QCEFF) tools are in the alpha release stage.
+The DART development team (dart@ucar.edu) would be happy to hear about your experiences and is
+anxious to build scientific collaborations using these new capabilities.
+
+To get started, make sure that you are on the quantile_methods branch of DART: 
+
+.. code-block:: text
+
+   git checkout quantile_methods
+
```

Comment:
These lines above (7-15) shoud be removed or edited before release. The tools will no longer be in the alpha release stage and the user will not have to checkout the quantile_methods branch

```
The Quantile-Conserving Ensemble Filter Framework (QCEFF) tools are in the alpha release stage.
The DART development team (dart@ucar.edu) would be happy to hear about your experiences and is
anxious to build scientific collaborations using these new capabilities.


To get started, make sure that you are on the quantile_methods branch of DART: 


.. code-block:: text


   git checkout quantile_methods

```

There is another comment like this at the very beginning of guide/qceff_probit.rst

Reply:
Since this branch is the live documentation, I'll remove the warnings just before merging.
I plopped them in the checklist:

![Screen Shot 2024-01-05 at 9 55 53 AM](https://github.com/NCAR/DART/assets/20047007/b6d21c83-8795-4776-85b0-8935996b71c9)

Thanks for the reminder (and future reminders so I/you/we don't miss this)